src/Raven.Server/Commercial/EmailValidator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Net.Mail;
+
+namespace Raven.Server.Commercial.LetsEncrypt;
+
+public class EmailValidator
+{
+    public static bool IsValidEmail(string email)
```

Comment:
Minor - It's already in `EmailValidator` so just `IsValid(string email)` should be enough

Reply:
done